### DISCUSSION TITLE
Global key events

### DISCUSSION THREAD

**Discussion by ju5t** [alpinejs/alpine#725] [4. General]
I would like to set a variable when '/' is pressed anywhere on a specific page to open a modal. I tried

```
x-on:keydown.slash
```

on the root element where AlpineJS is initialised, but it didn't work. I also tried to switch it to `escape` instead of `slash` but that didn't trigger it either.

Is this possible or do I want to do something that's not supported yet?

**Comment by ryangjchandler** [comment]
@ju5t If you're listening for a global event, you'll need to use the `.window` modifier.

**Reply by ju5t** [reply]
That works, thanks for the insanely fast help. Not sure how I missed that in the documentation.
